newIDE/app/src/InstancesEditor/InstancesRenderer/index.js

```diff
@@ -62,46 +64,47 @@ export default class InstancesRenderer {
     this.layersRenderers = {};
 
     this.pixiContainer = new PIXI.Container();
-    this.instanceMeasurer = {
-      getInstanceLeft: (instance: gdInitialInstance) => {
-        const layerName = instance.getLayer();
-        const layerRenderer = this.layersRenderers[layerName];
-        if (!layerRenderer) return instance.getX();
-
-        return layerRenderer.getInstanceLeft(instance);
-      },
-      getInstanceTop: (instance: gdInitialInstance) => {
-        const layerName = instance.getLayer();
-        const layerRenderer = this.layersRenderers[layerName];
-        if (!layerRenderer) return instance.getY();
-
-        return layerRenderer.getInstanceTop(instance);
-      },
-      getInstanceWidth: (instance: gdInitialInstance) => {
-        if (instance.hasCustomSize()) return instance.getCustomWidth();
 
+    this.temporaryRectangle = new Rectangle();
+    //TODO extract this to a class to have type checking (maybe rethink it)
```

Comment:
Yes, at least this could be simply a type (`type InstanceMeasurerInterface = {| getInstanceAABB: ... |};`) to ensure we follow the same interface. Ok not to do it now.